OsmAnd-java/src/net/osmand/map/OsmandRegions.java

```diff
@@ -695,11 +694,53 @@ private void sortSubregions(WorldRegion region, Comparator<WorldRegion> comparat
 		}
 	}
 
+	public List<WorldRegion> getWoldRegions(LatLon latLon) {
```

Comment:
This method doesn't look correct it and almost copy/paste findBinaryMapDataObject